tasks/fix-cat2.yml

```diff
@@ -40,54 +43,107 @@
-  command: "true"
-  changed_when: no
+  block:
+      - name: "MEDIUM | RHEL-07-010060 | PATCH | The Red Hat Enterprise Linux operating system must enable a user session lock until that user re-establishes access using established identification and authentication procedures."
+        blockinfile:
+            create: yes
+            dest: /etc/dconf/db/local.d/00-screensaver
+            block: |
+              [org/gnome/desktop/screensaver]
+              lock-enabled=true
+
+      - name: "MEDIUM | RHEL-07-010060 | PATCH | The Red Hat Enterprise Linux operating system must enable a user session lock until that user re-establishes access using established identification and authentication procedures."
+        command: dconf update
+        register: rhel_07_010060dconfout
+        changed_when: rhel_07_010060dconfout.rc <= 1
+        failed_when: rhel_07_010060dconfout.rc > 1
   when:
       - rhel7stig_gui
       - rhel_07_010060
   tags:
       - RHEL-07-010060
-      - notimplemented
 
 - name: "MEDIUM | RHEL-07-010061 | PATCH | The Red Hat Enterprise Linux operating system must uniquely identify and must authenticate users using multifactor authentication via a graphical user logon."
-  command: "true"
-  changed_when: no
+  block:
+      - name: "MEDIUM | RHEL-07-010061 | PATCH | The Red Hat Enterprise Linux operating system must uniquely identify and must authenticate users using multifactor authentication via a graphical user logon."
+        lineinfile:
+            create: yes
+            dest: /etc/dconf/db/local.d/00-defaults
+            insertafter: '^#[org/gnome/login-screen]'
+            regexp: '^#?enable-smartcard-authentication'
+            line: enable-smartcard-authentication=true
+
+      - name: "MEDIUM | RHEL-07-010061 | PATCH | The Red Hat Enterprise Linux operating system must uniquely identify and must authenticate users using multifactor authentication via a graphical user logon."
+        command: dconf update
+        register: rhel_07_010061dconfout
+        changed_when: rhel_07_010061dconfout.rc <= 1
+        failed_when: rhel_07_010061dconfout.rc > 1
   when:
       - rhel7stig_gui
       - rhel_07_010061
   tags:
       - RHEL-07-010061
-      - notimplemented
 
 - name: "MEDIUM | RHEL-07-010070 | PATCH | The Red Hat Enterprise Linux operating system must initiate a screensaver after a 15-minute period of inactivity for graphical user interfaces."
-  command: "true"
-  changed_when: no
+  block:
+      - name: "MEDIUM | RHEL-07-010070 | PATCH | The Red Hat Enterprise Linux operating system must initiate a screensaver after a 15-minute period of inactivity for graphical user interfaces."
+        lineinfile:
+            create: yes
+            dest: /etc/dconf/db/local.d/00-screensaver
+            insertafter: '^#[org/gnome/desktop/screensaver]'
+            regexp: '^#?idle-delay'
```

Comment:
I generally tend to prefer an `insertafter` the commented version and a `regexp` for the non-commented version, but not a huge deal.